source/release-notes.txt

```diff
@@ -10,6 +10,28 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.42.2
+----------------
+
+*Released March, 01 2024*
+
+New Features
+
+- Install updates without confirmation in the background by default
+  (:issue:`COMPASS-7616`)
+
+Bug Fixes
+
+- Double space not applied from schema or query history 
+  (:issue:`COMPASS-6980`)
+- Reset atlas search index on reopen and type change
```

Comment:
For whatever reason, https://github.com/mongodb-js/compass/commit/32f7985aa97090827588bf0e6b348185bbff0d2a does not have an associated COMPASS issue # in JIRA.